battery_monitor/src/battery_monitor.cpp

```diff
@@ -97,67 +97,93 @@ BatteryMonitor::BatteryMonitor(ros::NodeHandle &nodeHandle)
           diagnostic_tools::PeriodicMessageStatusParams{}
               .min_acceptable_period(minBatteryInfoCheckPeriod)
               .max_acceptable_period(maxBatteryInfoCheckPeriod)
-              .abnormal_diagnostic({diagnostic_tools::Diagnostic::WARN,
-                                    health_monitor::ReportFault::BATTERY_INFO_STALE})));
+              .abnormal_diagnostic(
+              {
+                diagnostic_tools::Diagnostic::WARN,
+                health_monitor::ReportFault::BATTERY_INFO_STALE
+              }
+)));
 
   diagnosticsUpdater.add(
       publisher_reportBatteryInfo.add_check<diagnostic_tools::MessageStagnationCheck>(
           "stagnation check",
           [batteryTotalCurrentSteadyBand](const battery_monitor::ReportBatteryInfo &a,
-                                          const battery_monitor::ReportBatteryInfo &b) {
+                                          const battery_monitor::ReportBatteryInfo &b)
+          {
             return (std::fabs(a.batteryPacksTotalCurrent - b.batteryPacksTotalCurrent) <
                     batteryTotalCurrentSteadyBand);
-          },
-          diagnostic_tools::MessageStagnationCheckParams{}.stagnation_diagnostic(
-              {diagnostic_tools::Diagnostic::WARN,
-               health_monitor::ReportFault::BATTERY_INFO_STAGNATED})));
+          }
+          , diagnostic_tools::MessageStagnationCheckParams{}
```

Comment:
@patapufate did `roslint` required the comma to be on the next line? It doesn't make the code more readable.

Reply:
No. nit error. Fixed